### PR TITLE
Smallpatches

### PR DESCRIPTION
**Pull request by InfusOnWoW**

### CODE REVIEW

File: WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasNewButton.lua [hunk 1]
```diff
@@ -1,7 +1,7 @@
 if not WeakAuras.IsLibsOK() then return end
 local AddonName, OptionsPrivate = ...
 
-local Type, Version = "WeakAurasNewButton", 26
+local Type, Version = "WeakAurasNewButton", 25
```
Comment (emptyrivers) [inline WeakAurasOptions/AceGUI-Widgets/AceGUIWidget-WeakAurasNewButton.lua:4]: version got anti-bumped

File: WeakAuras/Prototypes.lua [hunk 2]
```diff
@@ -703,29 +703,63 @@ if WeakAuras.IsClassicOrBCCOrWrath() then
 end
 
 if WeakAuras.IsRetail() then
-  function WeakAuras.CheckTalentSpellId(spellId)
+  local talentCheckFrame = CreateFrame("Frame")
```
Comment (mrbuds) [inline WeakAuras/Prototypes.lua:706]: Private.talentInfo update done in Types_Retail.lua should be merged with this

**Review by emptyrivers — COMMENTED** [review]
Rotate commit has TODO in commit message

File: WeakAuras/Prototypes.lua [hunk 3]
```diff
@@ -703,29 +703,63 @@ if WeakAuras.IsClassicOrBCCOrWrath() then
...
+            local node = C_Traits.GetNodeInfo(configId, nodeId)
+            if node.ID ~= 0 then
+              for _, talentId in ipairs(node.entryIDs) do
+                local entryInfo = C_Traits.GetEntryInfo(configId, talentId)
+                local definitionInfo = C_Traits.GetDefinitionInfo(entryInfo.definitionID)
+                local rank = node.activeRank
+                if node.activeEntry then
+                  rank = node.activeEntry.entryID == talentId and node.activeEntry.rank or 0
+                end
+                selectedTalents[definitionInfo.spellID] = rank
```
Comment (mrbuds) [inline WeakAuras/Prototypes.lua:734]: definitionInfo needs to be nil checked, got this error on /reload

```
2x WeakAuras/Prototypes.lua:734: table index is nil
[string "@WeakAuras/Prototypes.lua"]:734: in function <WeakAuras/Prototypes.lua:715>
```

**Comment by mrbuds** [comment]
I made a new texture with `Interface\AddOns\WeakAuras\Media\Textures\triangle-border.tga` editing "Allow Full Rotation" on/off and moving "Rotation" slider doesn't rotate the texture, but the aura button preview update.

https://i.imgur.com/Rin5oV7.gif

File: WeakAuras/Prototypes.lua (hunk 3 above)
Comment (mrbuds) [reply WeakAuras/Prototypes.lua:734]: mhm no my bad, error come spellID field that is nil

**Comment by InfusOnWoW** [comment]
Updated the PR, but haven't fixed all of your comments yet.

**Pull request merged**